src/Fixer/ClassNotation/PhpdocReadonlyClassCommentToKeywordFixer.php

```diff
@@ -58,7 +63,9 @@ class C {
                         EOT,
                     new VersionSpecification(8_02_00)
                 ),
-            ]
+            ],
+            null,
+            'If classes marked with `@readonly` annotation were extended anyway, applying this fixer may break the inheritance for these classes.'
```

Comment:
```suggestion
            'If classes marked with `@readonly` annotation were extended anyway, applying this fixer may break the inheritance for them.'
```

🤔? Somehow it looks weird when I read repeated "classes" now.